File: starting_filesets/C#/Roman Numerals/RomanNumeralsTest.cs

using NUnit.Framework;

[TestFixture]
public class RomanNumeralsTest
{
    [Test]
    public void Example()
    {
        int expected = 6 * 9;
        int actual = new RomanNumerals().Answer;
        Assert.AreEqual(expected, actual);
    }
}

File: starting_filesets/Java/Roman Numerals/RomanNumeralsTest.java

import org.junit.*;
import static org.junit.Assert.*;

public class RomanNumeralsTest 
{
    @Test
    public void example()
    {
        int expected = 6 * 9;
        int actual = new RomanNumerals().answer();
        assertEquals(expected, actual);
    }
}
